user/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.

```

Comment:
Consider adding test cases to this file to verify the functionality of your custom `User` model and any associated views or serializers. This will help ensure that your application behaves as expected and can handle edge cases.